tools/js-tools/tsconfig.base.json

```diff
@@ -9,6 +9,7 @@
 		"noEmit": true,
 		"resolveJsonModule": true,
 		"skipLibCheck": true,
-		"strict": false
+		"strict": false,
+		"target": "ESNext"
```

Comment:
This needed to be updated to fix this error which is caused by default `target` being `ES3`.


<img width="929" alt="Screenshot 2022-05-05 at 3 00 32 PM" src="https://user-images.githubusercontent.com/18226415/166905638-bffcbed4-f287-4e56-b90b-862087fb89a0.png">

Reply:
Thanks for that.